com.wings3d.WINGS.json

```diff
@@ -83,51 +94,50 @@
       "name": "rebar3",
       "buildsystem": "simple",
       "build-commands": [
-	"install -D rebar3 /app/bin/rebar3"
+        "install -D rebar3 /app/bin/rebar3"
       ],
       "sources": [
         {
           "type": "file",
-	  "url": "https://github.com/erlang/rebar3/releases/download/3.13.3/rebar3",
-	  "sha256": "a97e5368a9fce873a23c71c9bb3a63e760a7ace60842460e4704db5cad4df54c"
+          "url": "https://github.com/erlang/rebar3/releases/download/3.22.1/rebar3",
+          "sha256": "ac43aaae769650b021033a3be4709bec1d6abd965368d2c31c881daaf8dbbb07"
         }
       ]
     },
 
     {
-	"name": "opencl-headers",
-	"buildsystem": "simple",
-	"build-commands": [
+      "name": "opencl-headers",
+      "buildsystem": "simple",
+      "build-commands": [
 	    "cp -av CL /app/include/",
 	    "install -p -D -m 0644 LICENSE -t ${FLATPAK_DEST}/share/licenses/opencl-headers/"
-	],
+    ],
 
-	"sources": [
-	    { "type": "git",
-	      "url": "https://github.com/KhronosGroup/OpenCL-Headers.git",
-	      "commit": "c57ba81c460ee97b6b9d0b8d18faf5ba6883114b" /* v2020.06.16 */
-	    }
-	]
+      "sources": [
+          { "type": "git",
+            "url": "https://github.com/KhronosGroup/OpenCL-Headers.git",
+            "commit": "e049b16b5f157e2f28e7b5c301e71e1ccb3fe288"
+          }
+        ]
     },
 
     {
       "name": "cl",
       "buildsystem": "simple",
       "build-commands": [
-	/* OpenCL is optional and there is no headers in org.freedesktop.Sdk  20.08 */
-	"rebar3 compile || erlc -oebin src/*.erl && cp src/cl.app.src ebin/cl.app",
-	"install -d /app/lib/erlang/lib/cl/ebin /app/lib/erlang/lib/cl/priv /app/lib/erlang/lib/cl/include",
+        /* OpenCL is optional and there is no headers in org.freedesktop.Sdk  20.08 */
+        "rebar3 compile || erlc -oebin src/*.erl && cp src/cl.app.src ebin/cl.app",
+        "install -d /app/lib/erlang/lib/cl/ebin /app/lib/erlang/lib/cl/priv /app/lib/erlang/lib/cl/include",
         "install -t /app/lib/erlang/lib/cl/ebin ebin/*",
-	"install -t /app/lib/erlang/lib/cl/include include/*",
-	"[ -d priv ] && install -t /app/lib/erlang/lib/cl/priv priv/* || true"
+        "install -t /app/lib/erlang/lib/cl/include include/*",
+        "[ -d priv ] && install -t /app/lib/erlang/lib/cl/priv priv/* || true"
       ],
 
       "sources": [
         {
           "type": "git",
-	  "url": "https://github.com/tonyrog/cl.git",
-	  "tag": "cl-1.2.4",
-	  "commit": "f0a5631fdcf4e1c56e6bf1568c6c2c8fff0d66df"
+          "url": "https://github.com/tonyrog/cl.git",
+          "commit": "1f61da01ed3cd4b9aaec95ff352b1eeb64a63365"
         }
```

Comment:
We build with cl-1.2.4 on all other platforms, so this is untested.
Didn't even know there where changes in cl lib will take a look.

Reply:
> We build with cl-1.2.4 on all other platforms, so this is untested. Didn't even know there where changes in cl lib will take a look.

I saw he just applied some fixes, but I can let the previous one.